src/js/constants/cardano.js

```diff
@@ -9,24 +9,3 @@ export const NETWORK_IDS = Object.freeze({
     mainnet: 1,
     testnet: 0,
 });
-
-export const ADDRESS_TYPE = Object.freeze({
```

Comment:
I would rather leave the constants just to be sure we don't break anything. Maybe we can deprecate them - probably just by mentioning in the docs that these constants have been replaced by the enums and might be replaced in the future. I could add that in #794 after rebasing on top of this. Or is there a better way we could deprecate them?

Reply:
constants restored with comment (there is no mention about it in docs, feel free to update it)
enums aliased

95d8927a3f9d8c5a66eca0d9cd04126cb13eb51f

Reply:
additionally added same aliases in tests 6ed4de2a39f97ff65e0a99931087dc23c2de7a20